.gitignore

```diff
@@ -0,0 +1,3 @@
+test.md
+.hintrc
+.stylelintrc.json
```

Comment:
- [ ] Please add the node_modules to your .gitignore file.

Reply:
Hey @ShahierNashaat ! Updated the .gitignore file. Thank you.